v-next.sql

```diff
@@ -669,3 +669,582 @@ AND NOT EXISTS (SELECT * FROM openchpl.cqm_criterion cc INNER JOIN openchpl.cqm_
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (e)(1)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(e)(1)(i)(B)(2)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (e)(1)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(e)(1)(i)(C)(2)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (f)(3)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(f)(3)(ii)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (a)(4)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(a)(4)(iii)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (b)(1)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(b)(1)(i)(B)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (b)(1)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(b)(1)(i)(C)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (f)(3)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(f)(3)(i)(B)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (f)(7)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(f)(7)(i)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (a)(5)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(a)(5)(i)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (a)(8)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(a)(8)(iii)(B)(3)' AND  certification_edition_id = 2),
+	-1
+),
+(
+    (SELECT certification_criterion_id FROM openchpl.certification_criterion WHERE number = '170.314 (e)(1)'),
+    (SELECT test_functionality_id FROM openchpl.test_functionality WHERE number = '(e)(1)(i)(B)(1)(ii)' AND  certification_edition_id = 2),
+	-1
+);
+-- END OCD-2532
+----------------------------------------------------
+
+--re-run grants
+\i dev/openchpl_grant-all.sql
```

Comment:
It's helpful to have a newline at the end of the file so we can cat things together, but not required